lib/actions/removeMembership.js

```diff
@@ -0,0 +1,70 @@
+const loadTemplate = require('../loadTemplate')
+const config = require('../config')
+const getUrisFromReq = require('../getUrisFromReq')
+const sparql = require('../sparql/sparql')
+const getOwnedBy = require('../query/ownedBy')
+const pug = require('pug')
+
+module.exports = function (req, res) {
+  req.setTimeout(0) // no timeout
+
+  const { graphUri, uri, edit } = getUrisFromReq(req, res)
+
+  var memberUri = req.body.member
+  if (memberUri.startsWith('/public/')) {
+    memberUri = memberUri.replace('/public/', config.get('databasePrefix') + 'public/')
+  }
+  if (memberUri.startsWith('/user/')) {
+    memberUri = memberUri.replace('/user/', config.get('databasePrefix') + 'user/')
+  }
+
+  var templateParams = {
+    uri: uri,
+    memberUri: memberUri
+  }
+
+  var removeQuery = loadTemplate('sparql/removeMembership.sparql', templateParams)
+
+  console.log(removeQuery)
+
+  return getOwnedBy(uri, graphUri).then((ownedBy) => {
```

Comment:
Same comment as add -- can we check ownership earlier?

Reply:
Not sure where you have in mind.  The parts above are just defining the removeQuery.